docs/release-notes/rule-updates.mdx

```diff
@@ -12,6 +12,73 @@ import MoreHelp from "/src/components/MoreHelp"
 
 Welcome to monthly rule updates. This document includes selected new rules, removed false positives (FP), and other rule updates. These new rules and their updates are made by the Semgrep community and r2c.
 
+## January 2023
+
+### Community rules
+
+#### New rules from Semgrep community and r2c
+
+- Thanks [johnssimon007](https://github.com/johnssimon007)! New rule for empty encryption key in Python: <LinkToRegistryRule ruleId="python.cryptography.security.empty-aes-key.empty-aes-key" />
+- Thanks [johnssimon007](https://github.com/johnssimon007)! New rule for header injection in Python/Flask: <LinkToRegistryRule ruleId="python.flask.security.audit.host-header-injection-python.host-header-injection-python
+" />
+- Additional rule for deserialization vulnerabilities in Ruby: <LinkToRegistryRule ruleId="ruby.lang.security.bad-deserialization-yaml-env.bad-deserialization-yaml-env" />
```

Comment:
@p4p3r not working link

Reply:
Fixed in c5a3bb12c8d183b0b3a743d73ddc9a95d66fd377

Reply:
Fixed in https://github.com/returntocorp/semgrep-docs/pull/893/commits/c5a3bb12c8d183b0b3a743d73ddc9a95d66fd377